full-service/src/db/encryption_indicator.rs

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) 2020 MobileCoin Inc.
+
+//! DB impl for the Locked model.
+//!
+//! There are two "locked" states for a DB, in order to support a mode in which you can
+//! still submit transactions without accounts (and therefore without a password).
```

Comment:
Can you say more about what the idea here is? For example, when/how would a user submit transactions without access to the accounts stored in the wallet?